core-blocks/latest-comments/edit.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	Disabled,
+	PanelBody,
+	RangeControl,
+	ToggleControl,
+	ServerSideRender,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	BlockAlignmentToolbar,
+	BlockControls,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies.
+ */
+import './editor.scss';
+
+const MIN_COMMENTS = 1;
+const MAX_COMMENTS = 100;
+
+class LatestComments extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.setAlignment = this.setAlignment.bind( this );
+		this.setCommentsToShow = this.setCommentsToShow.bind( this );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+	}
+
+	toggleAttribute( propName ) {
```

Comment:
This function doesn't toggle anything on its own, it creates a function which performs the toggle. More accurately, it might be named `createToggleAttribute`.